app/javascript/controllers/amcharts_xy_chart_controller.js

```diff
@@ -0,0 +1,41 @@
+import { Controller } from 'stimulus'
```

Comment:
I am not sure if this is being used..

Reply:
Let's run an audit after we're live and get rid of all unused artifacts.